### PR TITLE
Add support for GraphQL 15

### PR DESCRIPTION
**Pull request by jesse-savary**
Updated the plugin to support `graphql@15.3.0`. Along the way I upgraded any dependencies that were complaining about peer issues and also fixed a test that was silently failing.

Let me know if anything needs to be fixed -- I'm always unsure when it comes to "updating" tests of any kind.

### CODE REVIEW

**Comment by jesse-savary** [comment]
Not sure what's up with codecov, coverage shouldn't be decreasing with these changes.

File: src/schema-manager/file-schema-manager.test.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { GraphQLSchema, graphql, buildSchema } from 'graphql';
-import { introspectionQuery } from 'graphql/utilities';
+import { getIntrospectionQuery } from 'graphql/utilities';
```
Comment (Quramy) [inline src/schema-manager/file-schema-manager.test.ts:2]: Thx.  I think this change of the function name does not backward compatibility. So I will release ts-graphql-plugin@v2.0.0, which requires `graphql >= 15.x` .

**Comment by Quramy** [comment]
@jesse-savary Thanks for your contribution and LGTM!

**Comment by Quramy** [comment]
I've release it as `ts-graphql-plugin@2.0.0`

**Pull request merged**